src/components/countries/country/Country.test.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable testing-library/no-container */
+/* eslint-disable testing-library/no-node-access */
+import { render } from '@testing-library/react';
+import React from 'react';
+import { Country } from './Country';
+
+describe('Test Country component', () => {
+  it('should render Country element by class', () => {
+    const { container } = render(<Country />);
+    expect(container.getElementsByClassName('country')[0]).toBeInTheDocument();
+    expect(container.getElementsByClassName('country__block')[0]).toBeInTheDocument();

```

Comment:
I think for this component it's more important to check whether Flag and CountryCardContent are rendered instead of wrapping divs.